### ISSUE TITLE
ExcelFile on Windows not closing appropriately

### ISSUE THREAD

**Issue by Mikuana** [Mikuana/rumydata#58]
Tests on Windows fail because the ExcelFile class that are opens files do not get appropriately closed before the test ends, which results in a exception. This appears to be related to the `read_only` property. An [issue](https://foss.heptapod.net/openpyxl/openpyxl/-/issues/1450) has been opened on the `openpyxl` repo, and a related [PR](https://foss.heptapod.net/openpyxl/openpyxl/-/merge_requests/388) exists but has not yet been merged.

**Comment by Mikuana** [comment]
@snotpus This issue is caused by an external dependency in a project, which has an imminent fix coming. Will wait for that to be addressed, and will just ignore windows tests until then.

**Comment by Mikuana** [comment]
Once issue is resolved, add `openpyxl` version with fix to the package dependencies, and add Windows OS to GitHub test actions.

```
# python_versions_test.yml

    runs-on: ${{ matrix.os }}
    strategy:
      max-parallel: 4
      matrix:
        python-version: [3.7, 3.8]
        os: [ubuntu-latest, windows-latest, macos-latest]
```

**Comment by Mikuana** [comment]
This community PR was rejected, but a fix has been incorporated into the default branch. This [commit](https://foss.heptapod.net/openpyxl/openpyxl/-/commit/70973bb7312a963a5b45b9d403f19a4ce4692edf) tracks the actual fix. Will wait for a package release to test.

**Comment by Mikuana** [comment]
The release of `openpyxl=3.0.6` was supposed to correct read-only files being left open on Windows, but tests are still failing. Should consider disabling `read_only` usage in `rumydata` until the feature stabilizes.

@Snotpus what are your thoughts on this.

**Comment by Snotpus** [comment]
That should be fine, I was attracted to trying `read_only` to speed up some
of my local testing, which it seemed to do and the notion of using it made
perfect sense for the context of rumydata! But it's not a necessity as far
as I can tell :)

**Comment by Mikuana** [comment]
Closed by #61
